providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/grpc/GrpcConnector.java

```diff
@@ -129,10 +129,6 @@ private void observeEventStream() {
 
             ServiceGrpc.ServiceStub localServiceStub = this.serviceStub;
 
-            if (this.streamDeadlineMs > 0) {
-                localServiceStub = localServiceStub.withDeadlineAfter(this.streamDeadlineMs, TimeUnit.MILLISECONDS);
-            }
-
```

Comment:
This for some reason, breaks the reconnect logic in e2e tests. I have no idea why. I'm not sure if it's a bug in gRPC, or elsewhere. I can't make any sense of it.